docs/interactions/Slash_Commands.md

```diff
@@ -627,6 +627,46 @@ And, done! The JSON looks a bit complicated, but what we've ended up with is a s
 
 ![A command with grouped subcommands and parameters. It says /permissions user get with arguments for a user and a channel.](command-with-groups-subcommands-parameters.png)
 
+## Permissions
+
+Need to keep some of your Slash Commands safe from prying eyes, or only available to the right people? Slash Commands support permission overwrites for all your commands. For both guild _and_ global commands, you can enable or disable a specific user or role in a guild from using a command.
+
+> info
+> For now, disabled commands still show up in the command picker, but are unable to be used.
```

Comment:
It might be better to clear up that it'll be unable to be used by people who don't have the permissions, but not everyone.